app/src/styles.sass

```diff
@@ -152,18 +153,27 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
 @include angular-material-typography($itero-mat-typography)
 
 
-.sidemenu
-  border-left:1px solid #704f6a
-  border-top: 1px solid #704f6a
-  border-bottom: 1px solid #704f6a
+
+.info-panel
+  border: 1px solid #704f6a
+  border-right: none
   background-color: #d5b6d0
-  float: right
-  height: 100%
-  margin-right: -16px
-  margin-left: 3px
-  box-shadow: -2px 0px 2px 0 rgba(255, 204, 244, 0.9)
-  .helpbutton
-    text-align: center
+  box-shadow: -5px 0px 5px 0 rgba(255, 204, 244, 0.7)
+  width: 80%
+  position: absolute
+  right: 0
+  display: flex
+  flex-flow: row nowrap
+  @media (min-width: $breakpoint-tablet)
+    width: 50%
+  @media (min-width: $breakpoint-laptop)
+    width: 30%
```

Comment:
On screens with width greater than 1000px, we won't use the information panel. Hence these lines are of little use and will introduce useless complications. Please remove them.